### DISCUSSION TITLE
model.predict

### DISCUSSION THREAD

**Discussion by Wearant** [ultralytics/ultralytics#510] [Q&A]
Please tell me how to get data about the class that was detected ?
Why names is returned empty

![DJs3HVa - Imgur](https://user-images.githubusercontent.com/78465917/213544086-eb8192d1-0c1d-4798-96cd-ca7f94f8b8e7.png)

**Comment by Wearant** [comment]
The issue is resolved

**Reply by glenn-jocher** [reply]
@Wearant great to hear that you've resolved the issue! If you have any more questions or run into any other challenges, feel free to reach out. Happy coding! 😊👍
